### DISCUSSION TITLE
Add ids on elements within Rainbowkit for automated testing

### DISCUSSION THREAD

**Discussion by addmorebass** [rainbow-me/rainbowkit#1451] [Ideas]
As one of the few automated testers within Web3, I propose we add IDs on each wallet type for better test automation.

For instance, Metamask would have id="metamask". This allows me to better automate testing my dApp.

**Comment by DanielSinclair** [comment]
@addmorebass This actually does exist! The Rainbow button as an example will have `rk-wallet-option-rainbow` set as the `data-testid`. It's camelcase, so MetaMask would look like `rk-wallet-option-metaMask`. Cheers 🍻
